
import unittest

from .utilities import reset

class TestCp(unittest.TestCase):
    @reset
    def test_cp(self):
        self.fail("This is not covered")
